doc/source/structures/volumes_and_files/volume.rst

```diff
@@ -34,27 +34,31 @@ Represents a :struct:`kOSProcessor` hard disk or the archive.
 
         * - :attr:`FILES`
           - :struct:`Lexicon`
-          - Lexicon of all files on the volume
+          - Lexicon of all files and directories on the volume
 
         * - :attr:`POWERREQUIREMENT`
           - :ref:`scalar <scalar>`
           - Amount of power consumed when this volume is set as the current volume
```

Comment:
*This may be something to refactor later, not as part of this PR:*

Is it really true that the electric charge is only consumed for the current volume?  That would seem a bit arbitrary and weird to me, given that with the new subdirectories changes in this PR, it will now be possible to use a volume's files by full path, without ever switching to that volume.  The fact that a volume isn't the current one has nothing to do with whether or not the files on it are being accessed.  Not anymore.  Maybe we might have some sort of ability to have a script shut a volume off when not in use.  When it's off, then trying to open its files doesn't work until you turn it on.  When it's off it wouldn't drain charge.  When it's on it would (regardless of if it's the current volume or not).